Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM node:18
 WORKDIR /app
+RUN npm i -g @nestjs/cli
```

Comment:
You shouldn't need to install the NestJS command-line tools on Docker: you only need that locally in order to run Nest commands line `nest new`, `nest generate`, etc.